dexs/aden/index.ts

```diff
@@ -0,0 +1,66 @@
+import { FetchResultVolume, SimpleAdapter } from "../../adapters/types";
+import fetchURL from "../../utils/fetchURL";
+import { CHAIN } from "../../helpers/chains";
+
+interface FuturesMarketRow {
+  symbol: string;
+  "24h_amount": string;
+  "24h_volume": number;
+  open_interest: number;
+  mark_price: number;
+}
+
+interface FuturesMarketResponse {
+  success: boolean;
+  data: {
+    rows: FuturesMarketRow[];
+  };
+}
+
+const fetch = async (timestamp: number): Promise<FetchResultVolume> => {
+  try {
+    // Using ADEN's CMC API endpoint with broker_id parameter
+    const response: FuturesMarketResponse = await fetchURL(
+      "https://api.orderly.org/v1/public/futures_market?broker_id=aden"
+    );
+
+    if (!response.success || !response.data?.rows) {
+      throw new Error("Invalid response from ADEN API");
+    }
+
+    // Calculate total 24h volume (already in USD)
+    const dailyVolume = response.data.rows.reduce((total, row) => {
+      return total + parseFloat(row["24h_amount"] || "0");
```

Comment:
whats the difference between 24h_amount & 24h_volume? 

we still have the issue of counting both sides of the trade, we are not doing this for other protocols